lib/importproject.cpp

```diff
@@ -792,13 +844,87 @@ bool ImportProject::importVcxproj(const std::string &filename, std::map<std::str
             }
             fsSetDefines(fs, fs.defines);
             fsSetIncludePaths(fs, Path::getPathFromFilename(filename), toStringList(includePath + ';' + additionalIncludePaths), variables);
+            for (const auto& path : sharedItemsIncludePaths) {
+                fs.includePaths.emplace_back(path);
+            }
             fileSettings.push_back(std::move(fs));
         }
     }
 
     return true;
 }
 
+static std::string stringReplace(const std::string& original, const std::string& toReplace, const std::string& replaceWith) 
```

Comment:
I am pretty sure we have at least one existing "replace string" implementation. Will check and pull that out into the utils so it can be re-used.

Reply:
I found one implementation and will make it available via the `utils.{cpp|h}` in a PR. I also need this for some changes I am working on.

Reply:
See #6325.